src/components/Header/Header.css

```diff
@@ -7,8 +7,8 @@
   text-transform: uppercase;
   background-color: var(--main-bg-color);
   font-family: "Montserrat", sans-serif;
```

Comment:
you can make this a custom variable if you use the same background colours

Reply:
Hey Hugh, many thanks for reviewing. Cool that you took the time. I will change this.